### ISSUE TITLE
Invoke method doesn't work with single quote & line translation

### ISSUE THREAD

**Issue by mk-initflow** [adamhartford/SwiftR#33]
When I send a string with single quote (') or line translation (\n) as an argument into invoke  method, it doesn't return anything
```
let message = "FirsLine\n SecondLine\n SingleQuote' "
hub!.invoke("sendMessage", arguments: [message]){ (result, error) in
      print("SendMessage result: \(result)  error: \(error)")
}
```

I guess that in that case we'll get syntax error in JavaScript
Because I'm debugging invoke method in SwiftR.swift & variable «js» has this value
```
SwiftR.swift:476 connection.runJavaScript(js)
js	String	"ensureHub('chathub').invoke('sendMessage', 'FirsLine\n SecondLine\n SingleQuote' ').done(function() { postMessage({ message: 'invokeHandler', hub: 'chathub', id: '7DBE8B14-D170-4CB9-8332-3A29E96E0374', result: arguments[0] }); }).fail(function() { postMessage({ message: 'invokeHandler', hub: 'chathub', id: '7DBE8B14-D170-4CB9-8332-3A29E96E0374', error: processError(arguments[0]) }); })"	
```

Any ideas on how to resolve the issue?
Thanks

**Comment by adamhartford** [comment]
Thanks for the report. Yeah, I think I can fix that. I'll keep you posted.

**Comment by adamhartford** [comment]
Should be all set now. I'll do some more testing myself before publishing to Cocoapods, but let me know if you run into any issues.

**Comment by adamhartford** [comment]
Fixed in v0.10.1
